examples/example7a_SAS/example7a.py

```diff
@@ -0,0 +1,49 @@
+# SPDX-FileCopyrightText: 2022 UChicago Argonne, LLC
+# SPDX-License-Identifier: MIT
+
+#########################################################
```

Comment:
We may want also at the next meeting to discuss the documentation of the example. I like your approach, but it makes the README irrelevant. Let's discuss which approach implement for all cases.

Reply:
If we do keep this, I would make it a proper docstring:
```Python
"""
Example problem ...
"""
```